tests/test_raster_import_pixellimit.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+#######
+# actinia-core - an open source REST API for scalable, distributed, high
+# performance processing of geographical data that uses GRASS GIS for
+# computational tasks. For details, see https://actinia.mundialis.de/
+#
+# Copyright (c) 2023 Lina Krisztian and mundialis GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#######
+
+"""
+Tests: Import raster with pixellimit check
+"""
+import unittest
+from flask.json import dumps as json_dumps
+
+try:
+    from .test_resource_base import (
+        ActiniaResourceTestCaseBase,
+        URL_PREFIX,
+        additional_external_data,
+    )
+except Exception:
+    from test_resource_base import (
+        ActiniaResourceTestCaseBase,
+        URL_PREFIX,
+        additional_external_data,
+    )
+
+__license__ = "GPLv3"
+__author__ = "Lina Krisztian"
+__copyright__ = "Copyright 2023, mundialis GmbH & Co. KG"
+__maintainer__ = "mundialis GmbH & Co. KG"
+
+
+class ImportRasterLayerPixellimitTestCase(ActiniaResourceTestCaseBase):
+    location = "nc_spm_08"
+    tmp_mapset = "mapset_rasterimport_pixellimit"
+    endpoint = f"/locations/{location}/mapsets/{tmp_mapset}/processing_async"
+    rimport_inp = "elevation"
+    # import resolution with which the process should fail:
+    rimport_res_fail = 0.1
+
+    def setUp(self):
+        # create new temp mapset
+        super(ImportRasterLayerPixellimitTestCase, self).setUp()
+        self.create_new_mapset(self.tmp_mapset, location_name=self.location)
+
+    def tearDown(self):
+        # delete mapset
+        self.delete_mapset(self.tmp_mapset, location_name=self.location)
+        super(ImportRasterLayerPixellimitTestCase, self).tearDown()
+
+    def test_pixellimit_not_reached(self):
+        """
+        Test import of raster, for which pixellimit is not reached
```

Comment:
Suggestion to remove the negation (my first thought was that the test is meant to fail because of that even):
```suggestion
    def test_pixellimit_allowed(self):
        """
        Test import of raster, for which pixellimit is not reached and therefore allowed
```